### PR TITLE
Fix typos in i18n directory

### PR DESCRIPTION
**Pull request by liticer**

### CODE REVIEW

**Comment by maciaszczykm** [comment]
It should be changed in HTML too.

**Comment by liticer** [comment]
@maciaszczykm which file? Sorry, I have not found any typos for these words except files in i18e directory.

**Comment by maciaszczykm** [comment]
True. Our current system fails to update `desc` values. We should keep them aligned. Take a look at:

https://github.com/kubernetes/dashboard/search?utf8=%E2%9C%93&q=Probed+at+&type=

We will use Angular translation system in the future.

**Comment by liticer** [comment]
@maciaszczykm I am sure that this commit includes all typos for these words in this repo, as I searched the entire code repo. So what should I do next?

File: i18n/messages-en.xtb [hunk 1]
```diff
@@ -1,12 +1,12 @@
...
   <translation id="5788105450172886273" key="MSG_ABOUT_ABOUT_3" desc="This is a label for button displayed on about page used to send feedback to GitHub new issue.">Send feedback</translation>
   <translation id="4249662097435421699" key="MSG_ABOUT_ABOUT_4" desc="This is the copyright statement displayed on the about page">Copyright 2015-{{ $ctrl.latestCopyrightYear}} The Kubernetes Authors.</translation>
   <translation id="4171916442768740608" key="MSG_ABOUT_ABOUT_5" desc="This is the creator reference on the footer of the about page"> Kubernetes Dashboard is made possible by the Dashboard &lt;a href="https://github.com/kubernetes/dashboard/graphs/contributors"&gt;community&lt;/a&gt; as an &lt;a href="https://github.com/kubernetes/dashboard"&gt;open source project&lt;/a&gt;. </translation>
-  <translation id="4426231252111230009" key="MSG_ABOUT_ABOUT_6" desc="Dashbord moto on the about page, just below the header">General-Purpose Web UI for Kubernetes Clusters</translation>
+  <translation id="4426231252111230009" key="MSG_ABOUT_ABOUT_6" desc="Dashboard moto on the about page, just below the header">General-Purpose Web UI for Kubernetes Clusters</translation>
```
Comment (maciaszczykm) [inline i18n/messages-en.xtb]: Change it to use values from the template:

https://github.com/kubernetes/dashboard/blob/41a1861c39b727278b7a1166b8ee9fccabd108d9/src/app/frontend/about/about.html#L28

**Comment by maciaszczykm** [comment]
Closing as stale. Please reopen after update.

**Pull request closed without merging**